### ISSUE TITLE
Do follow up PR to #2473

### ISSUE THREAD

**Issue by tcharding** [rust-bitcoin/rust-bitcoin#2554]
Fix all 8 unresolved discussions in #2473

**Comment by Kixunil** [comment]
Would you mind adding references to the specific comments? (I'd do it myself but I'm low on time and would rather do more important stuff now.)

**Comment by apoelstra** [comment]
Indeed -- presumably before #2473 merges there will be no "unresolved discussions" so this issue won't make sense at all.

**Comment by tcharding** [comment]
AFAIU we are merging with unresolved discussions if there is a link to an issue describing them - or have I got the wrong end of the stick again?

**Comment by apoelstra** [comment]
@tcharding no, my understanding was that we would resolve the discussion. But only after opening an issue.

If we were to leave the discussions open we wouldn't need an issue because we wouldn't be hiding the original conversations.

**Comment by tcharding** [comment]
i'm going to have to put some time into trawling back over everything we said and working out what we are up to, perhaps we can put it all together in #2564

**Comment by apoelstra** [comment]
Sure, will post there.
